Description: bump missing upstream, ensure migration and autopkgtest picks it
up properly.
Index: cargo/Cargo.toml
===================================================================
--- cargo.orig/Cargo.toml
+++ cargo/Cargo.toml
@@ -54,7 +54,7 @@ version = "0.4.1"
 version = "0.1.4"
 
 [dependencies.cargo-util]
-version = "0.2.6"
+version = "0.2.9"
 
 [dependencies.clap]
 version = "4.4.7"
